### PR TITLE
kernel: Make sure local versions.yaml file from runtime get precedence

### PR DESCRIPTION
**Pull request by ganeshmaharaj**
Fixes: #431

Signed-off-by: Ganesh Maharaj Mahalingam <ganesh.mahalingam@intel.com>

### CODE REVIEW

**Comment by ganeshmaharaj** [comment]
@jcvenegas @chavafg your thoughts? this is currently blocking me from testing https://github.com/kata-containers/runtime/pull/1504

File: scripts/lib.sh [hunk 1]
```diff
@@ -35,7 +35,11 @@ get_from_kata_deps() {
 	local branch="${2:-master}"
 	local runtime_repo="github.com/kata-containers/runtime"
 	GOPATH=${GOPATH:-${HOME}/go}
-	versions_file="versions-${branch}.yaml"
+  if [ -f ${GOPATH}/src/${runtime_repo}/versions.yaml ]; then
```
Comment (grahamwhaley) [inline scripts/lib.sh]: indentation broken ;-)

Comment (ganeshmaharaj) [reply scripts/lib.sh]: Fixed. :)

**Comment by ganeshmaharaj** [comment]
@jcvenegas per our conversation, i have updated the script with a comment and i will open a patch in tests to override the kernel version there.

**Comment by ganeshmaharaj** [comment]
@jcvenegas do we need to trigger CI for this patch given that it is just adding comments?

**Comment by jcvenegas** [comment]
@ganeshmaharaj I dont think so,
@devimc @grahamwhaley PTAL
just in case the merge button get blocked
/test

**Comment by grahamwhaley** [comment]
/test
just discussing this with @chavafg - yeah, we have to run the CI for everything to get the 'required' CIs OK'd before we can even force merge now... so...

**Comment by devimc** [comment]
the fix is a comment? :confused:

**Comment by ganeshmaharaj** [comment]
@devimc if you had glanced at the history of the change, you would have found https://github.com/kata-containers/packaging/commit/c0f3bb19d204d83e394bad5367bb0cd6930d671d which was the original patch. @jcvenegas later mentioned that such a change was previously there in this particular function and was removed so that during the time of release builds the kernel build will always check the master branch and pull changes from there, and moving this change to tests repository would be a good idea. To avoid another person in the future to come down the same pipe and run the same course, i have added a comment and submitted another patch to tests with https://github.com/kata-containers/tests/pull/1442

**Pull request merged**